tools/orca/src/generate.rs

```diff
@@ -55,7 +58,20 @@ pub async fn populate(_client: &KanidmOrcaClient, profile: Profile) -> Result<St
     // PHASE 1 - generate a pool of persons that are not-yet created for future import.
     // todo! may need a random username vec for later stuff
 
-    // PHASE 2 - generate persons
+    // PHASE 2 - generate groups for integration access, assign roles to groups
+    // actually these are just groups to decide what each person is supposed to do with their life
+    let groups = vec![
+        Group::new(
+            "readers".to_string(),
```

Comment:
Why not make this the Enum for the ActorRole? Because we'll need a role like "Ignore" so that groups can be created just for the purposes of membercship etc.

Reply:
Yeah actually I have no clue what I was smoking when I wrote that. Switched to the enum